static/elements/chromedash-upcoming.js

```diff
@@ -0,0 +1,95 @@
+import {LitElement, html} from 'lit-element';
+import '@polymer/iron-icon';
+import style from '../css/elements/chromedash-upcoming.css';
```

Comment:
I'd like to eventually phase out SASS and use lit-css instead.  Do you think that the CSS for each of these components could be specified in its .js file?  It's OK to still have a SASS file for the overall page for now.

Reply:
The CSS file for these components import certain variables and styles from other css file. It seems possible to phase out our use of SASS and use lit-css instead. However, It would be better to have separate PRs for this KR.

Reply:
ok